### PR TITLE
[CI:DOCS] Kube Play Doc: Document the support for K8S Secret

### PR DESCRIPTION
**Pull request by ygalblum**
#### Does this PR introduce a user-facing change?
No

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-kube-play.1.md.in [hunk 1]
```diff
@@ -122,6 +123,40 @@ spec:
 
 and as a result environment variable `FOO` will be set to `bar` for container `container-1`.
 
+`Kubernetes Secret`
+
+Kubernetes Secret represents a Podman named secret. The Kubernetes Secret is saved as a whole and may be referred as a source of environment variables or volumes in Pods or Deployments.
```
Comment (vrothberg) [inline docs/source/markdown/podman-kube-play.1.md.in:128]:
```suggestion
Kubernetes Secret represents a Podman named secret. The Kubernetes Secret is saved as a whole and may be referred to as a source of environment variables or volumes in Pods or Deployments.
```
?

**Review by vrothberg — COMMENTED** [review]
LGTM

Not sure about the nit.

**Comment by vrothberg** [comment]
@umohnani8 PTAL

File: docs/source/markdown/podman-kube-play.1.md.in (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman-kube-play.1.md.in:128]: @vrothberg is correct, the 'to' is needed there, seems like we've a grammar error that needs touching up elsewhere.  @ygalblum sorry about that, and would you mind fixing the other one while you're at it please?

**Comment by vrothberg** [comment]
Let's that in, @containers/podman-maintainers PTAL

**Pull request merged**